bundle/Resources/config/admin/controllers.yml

```diff
@@ -13,12 +13,16 @@ services:
         class: Netgen\TagsBundle\Controller\Admin\TagController
         parent: eztags.admin.controller.base
         public: true
+        lazy: true
         arguments:
             - "@eztags.api.service.tags"
             - "@ezpublish.api.service.content_type"
             - "@ezpublish.api.service.search"
             - "@translator"
             - "@eztags.pagerfanta.children_tags_adapter"
+            - "@eztags.pagerfanta.search_tags_adapter"
+        calls:
+            - [ setLanguages, [ "$languages$" ] ]
```

Comment:
Use config resolver injected in constructor instead of calling `setLanguages`. That way, the service does not need to be lazy. Besides, injecting config with setters does not work in eZ 3 anyway.

Reply:
Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249